src/wlr_layout_ui/__init__.py

```diff
@@ -26,45 +51,38 @@ def main():
             print("")
             for p in profiles:
                 print(f" - {p}")
+        elif sys.argv[1] == "-m":
+            load()
+            current_uids = set(di.uid for di in displayInfo)
+            for key in sorted(profiles.keys()):
+                prof_uids = {p["uid"] for p in profiles[key]}
+                # check that the two sets have the same elements
+                if len(prof_uids & current_uids) == len(current_uids):
```

Comment:
shouldn't be `prof_uids == current_uids` ?

Reply:
True! I was thinking it in terms of set comparisons, and just went with what felt the most natural way of comparing sets...but that's a nice shortcut :)